mrblib/mruby-cli/setup.rb

```diff
@@ -15,6 +15,10 @@ def run
         write_file("Dockerfile", dockerfile)
         write_file("docker-compose.yml", docker_compose_yml)
 
+        create_dir_p("tasks")
+        write_file("tasks/package.rake", package_rake)
+        write_file("tasks/test.rake", test_rake)
```

Comment:
I like that, it's cleaner to copy a file that we use in both cases rather than copy pasting the code. :+1: